test/helpers/steps/assess/dynakube.go

```diff
@@ -36,7 +36,7 @@ func UpdateDynakube(builder *features.FeatureBuilder, testDynakube dynatracev1be
 func verifyDynakubeStartup(builder *features.FeatureBuilder, testDynakube dynatracev1beta1.DynaKube) {
 	if testDynakube.NeedsOneAgent() {
 		builder.Assess("oneagent started", oneagent.WaitForDaemonset(testDynakube))
-		builder.Assess("osAgent can connect", oneagent.OSAgentCanConnect(testDynakube))
+		builder.Assess("oneagent pods are ready", oneagent.PodsAreReady(testDynakube))
```

Comment:
I think the `oneagent.WaitForDaemonset` already does what the `oneagent.PodsAreReady`
so I think `oneagent.PodsAreReady` is no necessary

Reply:
just remove it